keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone_test.py

```diff
@@ -0,0 +1,107 @@
+
+
+class MobileNetV3BackboneTest(tf.test.TestCase, parameterized.TestCase):
+    def setUp(self):
+        self.input_batch = tf.ones(shape=(2, 224, 224, 3))
+
+    def test_valid_call(self):
+        model = MobileNetV3SmallBackbone(
+            include_rescaling=False,
+        )
+        model(self.input_batch)
+
+    def test_valid_call_with_rescaling(self):
+        model = MobileNetV3SmallBackbone(
+            include_rescaling=True,
+        )
+        model(self.input_batch)
+
+    @parameterized.named_parameters(
+        ("tf_format", "tf", "model"),
+        ("keras_format", "keras_v3", "model.keras"),
+    )
+    def test_saved_model(self, save_format, filename):
+        model = MobileNetV3SmallBackbone()
+        model_output = model(self.input_batch)
+        save_path = os.path.join(self.get_temp_dir(), filename)
+        model.save(save_path, save_format=save_format)
+        restored_model = keras.models.load_model(save_path)
+
+        # Check we got the real object back.
+        self.assertIsInstance(restored_model, MobileNetV3SmallBackbone)
+
+        # Check that output matches.
+        restored_output = restored_model(self.input_batch)
+        self.assertAllClose(model_output, restored_output)
+
+    @parameterized.named_parameters(
+        ("tf_format", "tf", "model"),
+        ("keras_format", "keras_v3", "model.keras"),
+    )
+    def test_model_backbone_layer_names_stability(self):
+        model = MobileNetV3SmallBackbone(
+            include_rescaling=False,
+        )
+        model_2 = MobileNetV3SmallBackbone(
+            include_rescaling=False,
+        )
+        layers_1 = model.layers
+        layers_2 = model_2.layers
+        for i in range(len(layers_1)):
+            if "input" in layers_1[i].name:
+                continue
+            self.assertEquals(layers_1[i].name, layers_2[i].name)
+
+    def test_create_backbone_model_with_level_config(self):
+        model = MobileNetV3SmallBackbone(
+            include_rescaling=False,
+            input_shape=[256, 256, 3],
+        )
+        levels = [3, 4]
+        layer_names = [model.pyramid_level_inputs[level] for level in [3, 4]]
+        backbone_model = get_feature_extractor(model, layer_names, levels)
+        inputs = tf.keras.Input(shape=[256, 256, 3])
+        outputs = backbone_model(inputs)
+        self.assertLen(outputs, 2)
+        self.assertEquals(list(outputs.keys()), [3, 4])
+        self.assertEquals(outputs[3].shape, [None, 32, 32, 512])
+        self.assertEquals(outputs[4].shape, [None, 16, 16, 1024])
+
+    @parameterized.named_parameters(
+        ("one_channel", 1),
+        ("four_channels", 4),
+    )
+    def test_application_variable_input_channels(self, num_channels):
+        model = MobileNetV3SmallBackbone(
+            input_shape=(None, None, num_channels),
+            include_rescaling=False,
+        )
+        self.assertEqual(model.output_shape, (None, None, None, 2048))
```

Comment:
Should be 576. See original test ([link](https://github.com/keras-team/keras-cv/blob/953ea45d89905d24e7f270bb8747f74e89bbd364/keras_cv/models/mobilenet_v3_test.py#L23)).

Reply:
Done